### PR TITLE
feat(Skeleton): noBorderRadius prop

### PR DESCRIPTION
**Pull request by sergivillar**

### CODE REVIEW

File: playroom/snippets.tsx [hunk 1]
```diff
@@ -672,7 +672,7 @@ const cardSnippets: Array<Snippet> = [
         name: 'MediaCard with Image',
         code: `
         <MediaCard
-            media={<Image src="https://picsum.photos/1200/1200" aspectRatio="16:9" />}
```
Comment (yceballost) [inline playroom/snippets.tsx:675]: just to avoid this change I let the space xD

File: src/__stories__/media-card-story.tsx [hunk 2]
```diff
@@ -58,7 +58,7 @@ export const Default: StoryComponent<Args> = ({
                 media === 'video' ? (
                     <Video src={VIDEO_SRC} aspectRatio="12:5" dataAttributes={{qsysid: 'video'}} />
                 ) : (
-                    <Image aspectRatio="16:9" src={IMAGE_SRC} />
+                    <Image aspectRatio="16:9" src={IMAGE_SRC} noBorderRadius />
```
Comment (yceballost) [inline src/__stories__/media-card-story.tsx:61]: Is it not necessary, right?

**Review by yceballost — APPROVED** [review]
everything seems to work perfect 🙌

**Review by atabel — APPROVED** [review]

**Pull request merged**